ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -211,34 +211,26 @@ def get_partition_details(self, entity: Table) -> Optional[TablePartitionConfig]
         Args:
             entity: table entity
         """
-        # Should remove this with https://github.com/open-metadata/OpenMetadata/issues/5458
-        if entity.serviceType != DatabaseServiceType.BigQuery:
-            return None
         entity_config: TableConfig = self.get_config_for_entity(entity)
         if entity_config:
             return entity_config.partitionConfig
 
-        if entity.tablePartition:
-            if entity.tablePartition.intervalType in {
-                IntervalType.TIME_UNIT,
-                IntervalType.INGESTION_TIME,
-            }:
-                try:
+        if hasattr(entity, "tablePartition"):
+            try:
+                if entity.tablePartition.intervalType == IntervalType.TIME_UNIT:
                     partition_field = entity.tablePartition.columns[0]
-                except Exception:
-                    raise TypeError(
-                        "Unsupported ingestion based partition type. Skipping table"
-                    )
-
+                elif entity.tablePartition.intervalType == IntervalType.INGESTION_TIME:
+                    if entity.tablePartition.interval == "DAY":
+                        partition_field = "_PARTITIONDATE"
+                    else:
+                        partition_field = "_PARTITIONTIME"
                 return TablePartitionConfig(
                     partitionField=partition_field,
```

Comment:
What do you think of doing `if ... return` vs `if ... else`?

Reply:
I think as the partition_field is the one being changed based on conditions, i had to keep if…elif and return based on the updated value of partition_field
if the partition_field by any chance, comes as none, it will show up as
`f"Unsupported partition type {entity.tablePartition.intervalType}. Skipping table"`

Reply:
You could do something like this
```python
    if entity.tablePartition.intervalType == IntervalType.TIME_UNIT:
                    return TablePartitionConfig(partitionField=entity.tablePartition.columns[0])
                if entity.tablePartition.intervalType == IntervalType.INGESTION_TIME:
                    if entity.tablePartition.interval == "DAY":
                        return TablePartitionConfig(partitionField="_PARTITIONDATE")
                    return TablePartitionConfig(partitionField="_PARTITIONTIME")
```

Reply:
ohh okay, got it.. will make that change